src/gui/utils.cpp

```diff
@@ -53,6 +53,7 @@
 #include <QWindow>
 
 #include "base/global.h"
+#include "base/logger.h"
```

Comment:
I would prefer that utilities don't use log directly but just return error to the caller.

Reply:
It is impossible to return error for async operation.

Reply:
It seems a bad app design when "app components" (like Logger) are used by free utility functions. Especially if such functions contain asynchronous logic. There is no guarantee that "app component" isn't used after its destruction.

Reply:
I see. I dropped the logger.